### PR TITLE
Function to check packet loss during Ping

### PR DESCRIPTION
**Pull request by TasmiyaNalatwad**
Returns True for 0% packet loss and False if packet loss occurs

Signed-off-by: Tasmiya Nalatwad <<tasmiya@linux.vnet.ibm.com>>

### CODE REVIEW

**Comment by TasmiyaNalatwad** [comment]
@abdhaleegit  @PraveenPenguin  @Naresh-ibm  @vaishnavibhat  As suggested the function is moved to utils.
Please find the results below

[root@ltcrain41-lp4 net]# avocado run --test-runner runner ethtool_test.py -m ethtool_test.py.data/ethtool_test.yaml
Error running method "pre_tests" of plugin "fetchasset": 'Subscript' object has no attribute 'id'
JOB ID : fa4970f1d4f6f81c657e57c083e7095bdb568213
JOB LOG : /root/avocado-fvt-wrapper/results/job-2022-04-20T02.34-fa4970f/job.log
(01/18) ethtool_test.py:Ethtool.test_ethtool;run-args-PrivateFlag-hbond-d062: CANCEL: Device Doesn't support Private flags (5.45 s)
(02/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Driver-hbond-0537: PASS (27.37 s)
(03/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Autonegotiation-hbond-d9fe: FAIL: -r failed (10.64 s)
(04/18) ethtool_test.py:Ethtool.test_ethtool;run-args-PauseParameter-hbond-544a: PASS (26.64 s)
(05/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Coalescing-hbond-e440: PASS (26.96 s)
(06/18) ethtool_test.py:Ethtool.test_ethtool;run-args-RegisterDump-hbond-dc29: PASS (26.26 s)
(07/18) ethtool_test.py:Ethtool.test_ethtool;run-args-EEPROMDump-hbond-73b0: PASS (111.86 s)
(08/18) ethtool_test.py:Ethtool.test_ethtool;run-args-RingParameter-hbond-6b99: PASS (25.99 s)
(09/18) ethtool_test.py:Ethtool.test_ethtool;run-args-ShowOffloadProtocolFeature-hbond-4e7a: PASS (26.14 s)
(10/18) ethtool_test.py:Ethtool.test_ethtool;run-args-AdapterInitiate-hbond-b9a7: PASS (30.98 s)
(11/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Statistics-hbond-a820: PASS (27.11 s)
(12/18) ethtool_test.py:Ethtool.test_ethtool;run-args-SelfTest-hbond-29d7: FAIL: -t failed (11.55 s)
(13/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Show-permaddr-hbond-9546: PASS (27.13 s)
(14/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Get-dump-hbond-7fcb: WARN: Test passed but there were warnings during execution. Check the log for details. (26.80 s)
(15/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Show-time-stamping-hbond-e656: PASS (26.89 s)
(16/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Get-phy-tunable-hbond-415b: WARN: Test passed but there were warnings during execution. Check the log for details. (26.63 s)
(17/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Show-channels-hbond-e45c: PASS (26.44 s)
(18/18) ethtool_test.py:Ethtool.test_ethtool;run-args-Set-channels-combined_channel-other_channel-hbond-f639: PASS (105.34 s)
RESULTS : PASS 13 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 2 | INTERRUPT 0 | CANCEL 1
JOB HTML : /root/avocado-fvt-wrapper/results/job-2022-04-20T02.34-fa4970f/results.html
JOB TIME : 597.41 s

Attaching job.log
[ethtool.zip](https://github.com/avocado-framework/avocado/files/8527344/ethtool.zip)

File: avocado/utils/network/interfaces.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 import os
 import shutil
 from ipaddress import ip_interface
+from avocado.utils import process
 
 from avocado.utils.distro import detect as distro_detect
```
Comment (PraveenPenguin) [inline avocado/utils/network/interfaces.py:26]: please consider moving 

`from avocado.utils import process` after this import statement

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]
@TasmiyaNalatwad can you please see if we can add self test case for this utility

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
As we are checking for packet loss, so if there is packet loss, return True, and if there is no packet loss return False

Rest all Looks good to me

File: avocado/utils/network/interfaces.py (hunk 1 above)
Comment (TasmiyaNalatwad) [reply avocado/utils/network/interfaces.py:25]: Done

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Comment by TasmiyaNalatwad** [comment]
@PraveenPenguin As there is no selftest for interface.py We can have a separate PR for selftest covering tests for all functions in interfaces.py.

**Pull request closed without merging**